modules/components/src/Aggs/AggsWrapper.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Could you use this in the range and term agg? If it doesn't happen now it will never happen :D

Reply:
done! :D had that as a todo for the AFV rewrite actually, but done anyway :)

Reply:
ahh fair :)